resources/plistbuddy.rb

```diff
@@ -1,34 +1,38 @@
 resource_name :plistbuddy
```

Comment:
Can we call this resource `plist`? `plistbuddy` hints at implementation details that are irrelevant.

Reply:
I thought about this, and I like the idea a lot. Thought I would want @ehanlon and @jazaval before moving forward with it. In a way, it might be nice to actually incorporate the three major tools that help users manipulate property lists programmatically (`/usr/libexec/PlistBuddy`, `/usr/bin/defaults`, and `/usr/bin/plutil`), depending on the type of Plist and/or what needs to be accomplished.

Reply:
My pipe dream is that a `preference` resource uses `/usr/libexec/PlistBuddy`, `/usr/bin/defaults`, or `/usr/bin/plutil` depending on what's required, rather than require the user to know about ByHost preferences or actual plist paths. This is very lofty, but I think changing the name to `plist` is a step in the right direction, rather than trying to equate cookbook resources with command-line utilities.

Reply:
@jazaval I think that's a good lofty goal. From a UX perspective, we don't care what tool is used to get the job done, we just know what behavior we want. Generically naming this resource gives us reasonable freedom to use the best tool for the job.